subxt/src/blocks/extrinsic_types.rs

```diff
@@ -668,7 +668,7 @@ impl<'a> ExtrinsicSignedExtensions<'a> {
 
     /// The tip of an extrinsic, extracted from the ChargeTransactionPayment signed extension.
     pub fn tip(&self) -> Option<u128> {
-        let tip = self.find("ChargeTransactionPayment")?;
+        let tip = self.find("ChargeAssetTxPayment")?;
```

Comment:
Actually, both this and ChargeTransactionPayment could be used and contain a tip. Only one should ever be used at a time, and the tip I think is encoded at the start of either, so you can decode the start bytes in the same way for both.

So I'd try finding both and use whichever you find first :)